transmit-lib/src/cmr/transmit/urs.clj

```diff
@@ -17,6 +17,10 @@
   [conn]
   (format "%s/oauth/token?grant_type=client_credentials" (conn/root-url conn)))
 
+(defn- user-info-url
+  [conn username]
```

Comment:
do we require docstrings for private functions?

Reply:
Just following the precedent here, these URL generation functions do not include docstrings.